src/commands/shared/dependencies.ts

```diff
@@ -49,7 +50,7 @@ export async function getDependencyPaths(
 
       process.logger?.error(errorMessage)
 
-      const unFoundDependencies = diff(dependencies, foundDependencies)
+      const unFoundDependencies = _.difference(dependencies, foundDependencies)
```

Comment:
We don't actually have `lodash` listed in the dependencies, so this might cause issues.
It's best if you install this module https://www.npmjs.com/package/lodash.difference and import it individually.